### DISCUSSION TITLE
I need to get the notifications about the Orders using Amazon SP API

### DISCUSSION THREAD

**Discussion by Karthick2299** [amzn/selling-partner-api-models#3737] [General]
Could you provide the steps to achieve this  using Amazon SQS ? 
What are permissions needed to get the subscriptions ?
explain it.
Thanks in Advance!

**Comment by ayoubAnbara** [comment]
Try to follow this [tutorial](https://youtu.be/6YU4SoZED3w)

**Reply by Karthick2299** [reply]
Thanks! @ayoubAnbara

**Comment by stefnats** [comment]
Also, [this guide](https://developer-docs.amazon.com/sp-api/docs/notifications-api-v1-use-case-guide) helps a lot

**Reply by Karthick2299** [reply]
Thanks! @stefnats
